lib/blockscore/collection/member.rb

```diff
@@ -93,7 +93,16 @@ def parent_saved?
       #
       # @api private
       def ids
-        parent.attributes.fetch(:"#{Util.to_plural(instance.class.resource)}", [])
+        parent.attributes.fetch(resource_name, [])
```

Comment:
Not something that needs to be resolve in this PR, but I wonder if we should freeze collections if modifying them doesn't result in changes being persisted via the API.